data-structures/arrays/python/array_1D_output_all.py

```diff
@@ -0,0 +1,49 @@
+# Isaac Computer Science
+# Usage licensed under the Open Government Licence v3.0
+
+
+def create_list():
+    """Create a list of words"""
+
+    # Declare an empty list
+    spelling_words = []
```

Comment:
If you wanted the adding the elements to the array to be more similar to the pseidocode, you could add a loop to initialise a size 10 list of empty strings and then add the words by index? But I appreciate you have also made a very sensible note about how Python lists are not arrays which also explains what's going on here.